test/integration/TestTakerTest.php

```diff
@@ -30,7 +30,8 @@
 use oat\taoTestTaker\models\TestTakerService;
 use core_kernel_classes_Resource;
 use core_kernel_classes_Class;
-
+use oat\generis\test\TestCase;
+use oat\taoLti\models\classes\user\UserService as UserService;
```

Comment:
You don't need the alias here, since UserService is already the base name of the class.